src/07_slices.py

```diff
@@ -12,16 +12,17 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print("Second Element:", a[1])
 
 # Output the second-to-last element: 9
-print()
+# print("With Len:", a[len(a)-2])
+print("Second to last:", a[4])
 
```

Comment:
You could also use pythons reverse operation to count backwards from the list, Ex: `a[-2:-1]`. This is a weird pattern to get used to for sure though